tests/repl-multiline-test-expected-output

```diff
@@ -1,15 +1,4 @@
->=> 
->=> 
->=> 
->=> 
->=> 
->=> ... ... ... ... 30.
->=> 
->=> ... ... 
->=> 
->=> (1, 1)
->=> 
->=> 
->=> 
->=> 3.
+>=> >=> >=> >=> >=> >=> ... ... ... ... 30.
+>=> >=> ... ... >=> >=> (1, 1)
+>=> >=> >=> >=> 3.
```

Comment:
It looks funny because of the way terminals work, I guess? This is what you see when you pipe in text, which isn't what you do in practice. Here's what a repl session looks like now:

```
>=> x = 1
>=> 1 + 1
2
>=>
```

Here's what it used to look like:

```
>=> x = 1

>=> 1 + 1
2
>=> 
```

Notice the extra line about `x = 1`, which doesn't produce any output. That's all that's changed.

Reply:
Makes sense! Looks good then